app/models/type.rb

```diff
@@ -72,9 +72,9 @@ def <=>(type)
     name <=> type.name
   end
 
-  def self.all
-    find(:all, order: 'position')
-  end
+  # def self.all
+  #  find(:all, order: 'position')
+  # end
```

Comment:
The comment was rather intended as:

Why do you comment out your code if you have git available? ;-)

If you intend to re-enable this piece later I'd at least have expected a comment like `HACK: I want to enable this, but ...`